pkg/apis/config/defaults.go

```diff
@@ -156,6 +157,11 @@ type Defaults struct {
 	// a containerConcurrency of 0 (i.e. unbounded).
 	AllowContainerConcurrencyZero bool
 
+	// If true the `enableServiceLinks` on the pod template will
+	// be forced to false.
+	// See: https://github.com/knative/serving/issues/8498.
+	DisableServiceLinks bool
```

Comment:
This should be `*bool` and we should only use it when specified.

Reply:
We can't really do that in CM without making example stanza ugly.